daml-assistant/daml-helper/BUILD.bazel

```diff
@@ -94,7 +94,7 @@ da_haskell_test(
     srcs = ["test/DA/Daml/Helper/Test/Tls.hs"],
     data = [
         "daml-helper",
-        "//ledger/sandbox:sandbox-binary",
+        "//ledger/sandbox-:sandbox-binary",
```

Comment:
```suggestion
        "//ledger/sandbox-classic:sandbox-binary",
```
?

Reply:
Hmm, I tried to, but more things needed changing. This is me backtracking.

Reply:
Reverted for the moment

Reply:
Sandbox-next binaries not replaced in this PR.

Reply:
doing that in https://github.com/digital-asset/daml/pull/12761